ietf-document/draft-0rtt-bdp/draft-kuhn-quic-0rtt-bdp-09.xml

```diff
@@ -277,10 +277,14 @@
 				<t>Session Ticket</t>
 			</section>
 
-			<section title="Validity of the BDP extension">
-				<t>Time</t>
-				<t>IP</t>
-				<t>Adress validation</t>
+			<section title="Applicability of the BDP extension">
+				<t>This section discusses that applicability of the BDP extension.</t>
+				<t>The following events could make the BDP extension inappropriate:<list style="symbols">
+					<t>IP address changes: If the client changes its IP address, it may exploit a different network to reconnect. The exploited path may not exhibit the same characteristics.</t> 
+					<t>Rate-limitation induced by QUIC: The proposed mechanisms would be limited by other rate-limitation mechanisms of QUIC, such as the address validation process that is used to prevent amplification attacks <xref target="I-D.ietf-quic-transport"></xref>.</t>
+					<t>Lifetime of the extension: Based on experimental feedback and to reduce resource utilization at the client, the BDP extension may be proposed with a lifetime that could be changed. This follows the idea of NewSessionTicket of TLS <xref target="RFC8446"></xref>.</t>
+				</list></t>
+				<t>That being said, exploiting the safety nets proposed in <xref target="safety_guide"></xref> would reduce any harm that could be caused by using inappropriate BDP parameters. This safety guidelines reduce the interests of the proposed approach, but limits the risks to add congestion in a congested network.</t>
```

Comment:
```suggestion
<t>That being said, exploiting the safety nets proposed in <xref target="safety_guide"></xref> would reduce any harm that could be caused by using inappropriate BDP parameters. The safety guidelines reduce the potential impact of the proposed approach, but limit the risks of adding congestion in a congested network.</t>
```

I'm not sure what the "interests" is referring to here. It seems like "impact" is what is meant?

Reply:
I meant 'benefits of the proposed approach' or 'gains brought by the proposed approach'. 
I guess one of the two is closer to what we meant - do you have any preferences ?

Reply:
I think 'gains brought by the proposed approach' is the most applicable here then

Reply:
Done in commit https://github.com/NicoKos/QUIC_HIGH_BDP/commit/aca23f3fe6fe4d62099febff4f3a35f411dc9c11